pandas/tests/groupby/test_function.py

```diff
@@ -223,13 +223,13 @@ def test_arg_passthru():
         tm.assert_index_equal(result.columns, expected_columns)
 
     expected_columns = Index(["int", "float", "string", "category_int", "timedelta"])
-    for attr in ["sum"]:
-        f = getattr(df.groupby("group"), attr)
-        result = f()
-        tm.assert_index_equal(result.columns, expected_columns_numeric)
 
-        result = f(numeric_only=False)
-        tm.assert_index_equal(result.columns, expected_columns)
+    f = getattr(df.groupby("group"), "sum")
+    result = f()
+    tm.assert_index_equal(result.columns, expected_columns_numeric)
+
+    result = f(numeric_only=False)
```

Comment:
same

Reply:
Done, I think it's a bit easier to read / understand if we just call these functions directly instead of assigning to f then calling, so changed that as well